src/main/java/com/hladchenko/springrestapi/service/UserService.java

```diff
@@ -0,0 +1,46 @@
+package com.hladchenko.springrestapi.service;
+
+import com.hladchenko.springrestapi.entity.Note;
+import com.hladchenko.springrestapi.entity.User;
+import com.hladchenko.springrestapi.excetion.UserNotFoundException;
+import org.springframework.stereotype.Service;
+
+import java.util.*;
+
+@Service
+public class UserService {
+    private static final Map<UUID, User> userMap = new HashMap<>();
+
+    public List<User> getUsers() {
+        return userMap.values().stream().toList();
+    }
+
+    public User getUser(UUID uuid) {
+        return userMap.get(uuid);
```

Comment:
I'd recommend to add check on user existence and warn a client if it's absent with proper error msg